docs/development/core/public/kibana-plugin-public.uisettingsstart.md

```diff
@@ -0,0 +1,12 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [kibana-plugin-public](./kibana-plugin-public.md) &gt; [UiSettingsStart](./kibana-plugin-public.uisettingsstart.md)
+
+## UiSettingsStart type
+
+
+<b>Signature:</b>
+
+```typescript
+export declare type UiSettingsStart = UiSettingsClient;

```

Comment:
@skaapgif Do you think we should export the `UiSettingsClient` class too so that it's documented?